views/home.tmpl

```diff
@@ -205,7 +237,7 @@
                         <div class="d-flex flex-table-row mempool-row">
                             <a class="hash truncate-hash keyboard-target" style="flex: 1 1 auto" href="/tx/{{.Hash}}" title="{{.Hash}}">{{.Hash}}</a>
                             <span style="flex: 0 0 65px" class="mono text-right ml-1">{{.Type}}</span>
-                            <span style="flex: 0 0 105px" class="mono text-right ml-1 fs13">
+                            <span style="flex: 0 0 105px" class="mono text-right ml-1 fs13 text-secondary">
```

Comment:
Your screenshot is showing a weird alignment on this "column".

Reply:
fixed, was messing around with the layout there, but ended up sticking with the inline-block approach